.github/workflows/e2e.yml

```diff
@@ -0,0 +1,54 @@
+# e2e test for mention-links
+
+name: CI for mention-links
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: 
+      - develop
+      - automation/playwright
```

Comment:
@alvitazwar Remove the develop and automation/playwright branch. As there is only one main(Master) branch present in repo.

Reply:
Resolved